client/src/transmitter/index.ts

```diff
@@ -66,21 +108,17 @@ async function initStreamData() {
   const { humanId, streamKey } = await res.json()
   console.log('streaming to', humanId)
 
-  _playbackId = humanId
-  _streamKey = streamKey
+  _playback_id = humanId
+  _stream_key = streamKey
 
-  const portStr = isLocalOrIp ? `:${port}` : ''
-  playbackUrl.innerText = `${protocol}//${hostname}${portStr}/${humanId}`
+  const portStr = is_local_or_ip ? `:${port}` : ''
```

Comment:
I wonder if this could simply be 
```
portStr = port ? `:${port}` : ''
```
and we drop that local or ip thing

Reply:
Sounds right.

Reply:
It makes the whole `isLocalOrIp` thing unnecessary!